src/components/ClassBlock.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC } from "react";
+import { ClassBlockProps } from "@/lib/types";
+
+const ClassBlock: FC<ClassBlockProps> = (props) => {
+  const typeClasses = {
+    W: "bg-red-300",
+    L: "bg-blue-300",
+    C: "bg-green-300",
+    S: "bg-orange-300",
+    P: "bg-fuchsia-200",
+  };
```

Comment:
fajne, ale raczej zdefiniuj to poza komponentem i rzuć `as const` na to, wtedy typescript będzie fajnie sprawdzał Ci typy
```suggestion
  const typeClasses = {
    W: "bg-red-300",
    L: "bg-blue-300",
    C: "bg-green-300",
    S: "bg-orange-300",
    P: "bg-fuchsia-200",
  } as const;
```